src/Psalm/Internal/ExecutionEnvironment/SystemCommandExecutor.php

```diff
@@ -22,6 +22,10 @@ final class SystemCommandExecutor
      */
     public function execute(string $command) : array
     {
+        if (!function_exists('exec')) {
+            throw new \RuntimeException(sprintf('Failed to execute command: %s', $command));
```

Comment:
the error message is misleading. It should point out the fact that `exec()` does not exist or similar